extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -189,30 +208,24 @@
                     connection.setParentIdentifier(LDAPAuthenticationProvider.ROOT_CONNECTION_GROUP);
                     connections.put(name, connection);
 
-                }
-
-                // Deal with issues following LDAP referrals
-                catch (LDAPReferralException e) {
```

Comment:
Does the new LDAP API not throw exceptions for referrals?

Reply:
Not sure why it got removed and did not get added back.  Added it back and cleaned up some of the other error messages.

Reply:
Added back.